.circleci/config.yml

```diff
@@ -1,14 +1,14 @@
 version: 2.1
 orbs:
-  ruby: circleci/ruby@2.1.0
+  ruby: circleci/ruby@2.3.1
 
 executors:
   # TODO: we don't need ES/PG/Redis for "lint" job, so we could create another
   # stripped-down executor if we want?
   libraries:
     working_directory: ~/libraries
     docker:
-      - image: ruby:3.2.5-slim-bullseye
+      - image: cimg/ruby:3.2-node
```

Comment:
I think we just needed the custom packages below for node, so this switches to the node image